### PR TITLE
Alina tinca/tema8

### PR DESCRIPTION
**Pull request by tincageorgianaalina**
Am adaugat tema 8

### CODE REVIEW

**Review by curtnazarEliz — APPROVED** [review]

**Review by ioanaandroflorea — APPROVED** [review]

File: src/test/resources/alinaTinca/suiteTema8Desktop.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<suite name="Homework8-Desktop Test Suite" verbose="1" >
```
Comment (95teo) [inline src/test/resources/alinaTinca/suiteTema8Desktop.xml:1]: Why are suite.xml and group.xml files empty? Please refer to #39 and put the content back inside them. Thanks!

File: src/test/java/alinaTinca/tests/Tema8/tests/JavaScriptRedirectsPageTests.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+public class JavaScriptRedirectsPageTests extends BaseTest{
+    @Test(groups = {"mobile", "desktop"})
+    public void clickOnFirstButton() {
+        driver.get("https://testpages.herokuapp.com/styled/javascript-redirect-test.html");
+        JavaScriptRedirectsPage javaScriptRedirectsPage = new JavaScriptRedirectsPage(driver);
+        System.out.println("The message in the paragraph on the first page is: " + javaScriptRedirectsPage.getParagraph());
+        javaScriptRedirectsPage.clickOnFirstButton();
+        JavaScriptRedirectsConfirmationPage javaScriptRedirectsConfirmationPage = new JavaScriptRedirectsConfirmationPage(driver);
+//        Assert.assertEquals(driver.getCurrentUrl(),
```
Comment (95teo) [inline src/test/java/alinaTinca/tests/Tema8/tests/JavaScriptRedirectsPageTests.java:24]: In order to assert urls, you'll usually have to wait for page to load.

File: src/test/java/alinaTinca/tests/Tema8/pageObjects/SimpleDynamicButtonConfirmationPage.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    @FindBy(id ="buttonmessage")
+    WebElement text;
+
+    //Create a method that display the message before opening all the buttons
+    public String displayTextBeforeClickAllButtons(){
+        return text.getText();
+    }
+
+    //Create a method that display the message after opening all the buttons
+    public String displayTextAfterClickAllButtons(){
```
Comment (95teo) [inline src/test/java/alinaTinca/tests/Tema8/pageObjects/SimpleDynamicButtonConfirmationPage.java:36]: This method duplicates displayTextBeforeClickAllButtons() method. You can delete one of them and keep only one an use it in your test whenever needed.

File: src/test/java/alinaTinca/tests/Tema8/pageObjects/RefreshPage.java [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+    //Create a method that return the current timestamp
+    public String getCurrentTimestamp(){
+//        wait.until(ExpectedConditions.invisibilityOfElementLocated(By.id("refreshdate")));
+//        System.out.println("Condition has been executed!");
+        return refreshButton.getText();
+    }
+
+    //Create a method that return the refreshed timestamp
+    public String getRefreshedTimestamp(){
```
Comment (95teo) [inline src/test/java/alinaTinca/tests/Tema8/pageObjects/RefreshPage.java:40]: This method duplicates getCurrentTimestamp() method. You can delete one of them and keep only one an use it in your test whenever needed.

File: src/test/java/alinaTinca/tests/Tema8/pageObjects/DisabledDynamicButtonConfirmationPage.java [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    @FindBy(id="buttonmessage")
+    WebElement text;
+
+    //Create a method that display the text before opening all buttons
+    public String displayTextBeforeClickAllButtons(){
+        return text.getText();
+    }
+
+    //Create a method that display the text after opening the all buttons
+    public String displayTextAfterClickAllButtons(){
```
Comment (95teo) [inline src/test/java/alinaTinca/tests/Tema8/pageObjects/DisabledDynamicButtonConfirmationPage.java:38]: This method duplicates displayTextBeforeClickAllButtons() method. You can delete one of them and keep only one an use it in your test whenever needed.

**Review by 95teo — CHANGES_REQUESTED** [review]
Please solve also your conflicts. Thanks!

File: src/test/resources/alinatinca/suite.xml [hunk 6]
```diff
@@ -1,11 +0,0 @@
-<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
```
Comment (95teo) [inline src/test/resources/alinatinca/suite.xml:1]: Please don't delete this suite.xml as it was used for Tema 7. Add it in the same directory with the one for desktop and mobile. Thanks!

File: src/test/java/alinaTinca/tests/Tema8/pageObjects/DisabledDynamicButtonConfirmationPage.java (hunk 5 above)
Comment (95teo) [reply src/test/java/alinaTinca/tests/Tema8/pageObjects/DisabledDynamicButtonConfirmationPage.java:38]: You still have two methods here. Choose only one to use in your tests and delete the other one. Thanks!

**Review by 95teo — CHANGES_REQUESTED** [review]

**Review by 95teo — APPROVED** [review]

**Pull request merged**